src/pages/Locations/LocationDetails/components/LocationPhotosGrid/LocationPhotosGrid.jsx

```diff
@@ -0,0 +1,197 @@
+import * as React from "react";
+import {useState} from "react";
+import {AppBar, Button, Dialog, Skeleton, Slide, Toolbar, Typography} from "@mui/material";
+import {ViewGridIcon} from "@heroicons/react/solid";
+import CloseIcon from '@mui/icons-material/Close';
+import './LocationPhotosGrid.css'
+import {Carousel} from "react-responsive-carousel";
+import "react-responsive-carousel/lib/styles/carousel.min.css";
+
+
+const placeholderImages = [
+    'https://via.placeholder.com/1080x900.png?text=Loading...',
+]
+const Transition = React.forwardRef(function Transition(props, ref) {
+    return <Slide direction="up" ref={ref} {...props} />;
+});
+const LocationPhotosGrid = (props) => {
+    const [gallery, setGallery] = useState(false)
+    const [loaded, setLoaded] = useState(false)
+    const handleGalleryOpen = () => {
+        setGallery(true)
+    }
+    const handleGalleryClose = () => {
+        setGallery(false)
+    }
+
+    return (
+        <div>
+            {props.photos == null ?
+                <div style={{display: 'flex', justifyContent: 'center'}}>
+                    <div className="w-4_10 lg-w-1_2 md-w-1 mr-5">
+                        <img className="mt-10 start-rounded md-end-rounded" alt="location"
+                             src={placeholderImages[0]}/>
+                    </div>
+                    <div className="alternative-photos lg-w-1_2 w-4_10 md-visible">
+                        <div className="w-1_2 lg-w-1 flex">
+                            <img className="mt-10 mr-5 lg-end-rounded" alt="location" src={placeholderImages[0]}/>
+                            <img className="lg-visible mt-10 top-end-rounded" alt="location"
+                                 src={placeholderImages[0]}/>
+                        </div>
+                        <div className="lg-visible w-1_2 flex">
```

Comment:
Codacy has a fix for the issue: [Delete `············`](https://app.codacy.com/gh/mohammadnedaei/Nearby-Locations-React/pullRequest?prid=9855911)
```suggestion
            <div className="lg-visible w-1_2 flex">
```